pycloudlib/oci/instance.py

```diff
@@ -168,6 +168,28 @@ def start(self, wait=True):
         if wait:
             self.wait()
 
+    def wait(self, **kwargs):
```

Comment:
I think it'd be better to modify the base class instead. Something like `def wait(self, timeout=40, **kwargs):` (with an updated docstring explaining why 40) that gets passed through to `_wait_for_execute()`. The current hardcoded-for-ec2 option isn't great.